app/views/admin/responses/_attachments.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if attachable.can_order_attachments? && attachable.attachments.many? %>
+  <p class="govuk-body"><%= link_to 'Reorder attachments', reorder_admin_response_attachments_path(@response), class: "govuk-link govuk-link--no-visited-state" %></p>
```

Comment:
I'm struggling to get the required path here to show the attachments that are added to a response.

Reply:
Ah i see whats going on. We're using polymorphism to determine which class the attachment belongs to. An attachment can belong to an `Edition` or a `Response`, both of which have different data models.

The same view is used in both contexts so it needs to be model agnostic if possible & if not it needs to account for the different data models. 

On this line https://github.com/alphagov/whitehall/blob/main/app/views/admin/attachments/reorder.html.erb#L1
it's trying to set the context to the attachables title. When that's an edition it's fine, but when it's an object that doesn't have  a title like a response it blows up.

We can change this line of code 

```
<% content_for :context, attachable.title %>
```

to 

```
<% content_for :context, attachable.try(:title) %>
```

so it retuns nil if aan object doesn't implement the title method.

Let me know if that makes no sense!

Reply:
Ah great, yes that makes sense.